3ds2/src/main/java/com/adyen/checkout/adyen3ds2/Adyen3DS2Configuration.kt

```diff
@@ -106,3 +107,33 @@ class Adyen3DS2Configuration private constructor(
         }
     }
 }
+
+fun CheckoutConfiguration.adyen3DS2(
+    configuration: Adyen3DS2Configuration.Builder.() -> Unit = {}
+): CheckoutConfiguration {
+    val config = Adyen3DS2Configuration.Builder(shopperLocale, environment, clientKey)
+        .apply {
+            amount?.let { setAmount(it) }
+            analyticsConfiguration?.let { setAnalyticsConfiguration(it) }
+        }
+        .apply(configuration)
+        .build()
+    addActionConfiguration(config)
+    return this
+}
+
+fun CheckoutConfiguration.getAdyen3DS2Configuration(): Adyen3DS2Configuration? {
```

Comment:
Do these need to be public?

Reply:
They don't, but I thought it would maybe nice to have for merchants